src/Controller/UserController.php

```diff
@@ -164,27 +205,40 @@ public function modifyUser(User $user, $userId, UserRepository $repository, Enti
         $manager->persist($registeredUser);
         $manager->flush();
 
-        return $this->view($registeredUser, Response::HTTP_ACCEPTED);
+        $keyAll = 'user.all?page=' . $request->query->getInt('page', 1);
+        $keyOnce = 'user.once';
+
+        $cacheAll = $this->adapter->getItem($keyAll);
+        $cacheOnce = $this->adapter->getItem($keyOnce);
+
+        if (true === $cacheAll->isHit()){
```

Comment:
idem içi, deux fois la même condition

Reply:
No, this action is applicated to 2 different keys